lib/src/_shared/DateTextField.jsx

```diff
@@ -48,6 +48,8 @@ export class DateTextField extends PureComponent {
     keyboard: PropTypes.bool,
     /** Icon displayed for open picker button in keyboard mode */
     keyboardIcon: PropTypes.node,
+    /** enables/disable automatic opening of the picker when the user clicks enter */
+    openPickerOnEnter: PropTypes.bool,
```

Comment:
I thought maybe it would be more convenient to name prop smth like `disableOpenOnEnter`.
It would be looks like
```jsx
<DatePicker disableOpenOnEnter />
```
instead of 
```jsx
<DatePicker openPickerOnEnter={false} />
```

Reply:
Yeah your right that is better, i will change it :)